### DISCUSSION TITLE
Приложить файл Figma для создания схем в фирменном стиле

### DISCUSSION THREAD

**Discussion by vitalybaev** [doka-guide/content#2741] [Идеи]
Когда я писал статью по GitLab CI, я нарисовал схему, которую потом перерисовали в фирменном стиле (шрифты/цвета и прочее).
Может быть где-нибудь в руководстве по стилю приложить ссылку на эталонный файл Figma, в котором есть примеры таких схем с нужными цветами, шрифтами и т.д.

И при добавлении своей схемы просить указывать ссылку на исходник схемы, чтобы её подправить, если требуется.

Даже несмотря на то, что часто результат нужно будет дорабатывать, я вижу в этом два преимущества:
1. Для редакторов Доки: абсолютно точно проще внести правки в уже существующий файл, чем перерисовывать полностью
2. Для контрибьюторов: наличие готовых строительных блоков для таких схем может упростить создание своих схем, что может положительно сказаться на конечном качестве статей (хорошая иллюстрация бывает критически важной).

**Comment by furtivite** [comment]
@vitalybaev, привет! Идея, безусловно, интересная. Гораздо удобнее, если автор сможет сразу сделать классную схему, и её не придётся править. Мы обсудим с редакцией и вернёмся сюда с ответом.

**Comment by solarrust** [comment]
Публичная фигма со схемами опубликована. [Инструкция по схемам](https://github.com/doka-guide/content/blob/main/docs/schemes.md)
